amethyst_network/src/simulation/transport/udp.rs

```diff
@@ -0,0 +1,154 @@
+//! Network systems implementation backed by the UDP network protocol.
+use crate::simulation::{
+    events::NetworkSimulationEvent,
+    guarantees::{DeliveryGuarantee, OrderingGuarantee},
+    message::Message,
+    resource::NetworkSimulationResource,
+    timing::{NetworkSimulationTime, NetworkSimulationTimeSystem},
+    transport::socket::Socket,
+};
+use amethyst_core::{
+    bundle::SystemBundle,
+    ecs::{DispatcherBuilder, Read, System, World, Write},
+    shrev::EventChannel,
+};
+use amethyst_error::Error;
+use bytes::Bytes;
+use log::{error, warn};
+use std::{
+    io,
+    net::{SocketAddr, UdpSocket},
+};
+
+/// Use this network bundle to add the various underlying UDP network systems to your game.
+pub struct UdpNetworkBundle {
+    recv_buffer_size_bytes: usize,
+}
+
+impl UdpNetworkBundle {
+    pub fn new(recv_buffer_size_bytes: usize) -> Self {
```

Comment:
just `recv_buffer_size`

Reply:
I really do like putting the units on the names when I can. That way everyone knows that this is the size in bytes

Reply:
Alright, can agree to that, however, buffers are most likely filled with bytes and no other types.